k8s/installation/observability.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Monitoring the Kubernetes Operator
+sidebarTitle: Observability
+description: Learn how to enable and customize ngrok's Kubernetes ingress Observability features.
+---
+
+Configuring metrics and logging for the ngrok Kubernetes Operator is essential for gaining visibility into how your ngrok ingress resources behave within your cluster. This guide explains the available observability features, how to enable and how to customize them. By setting up proper metrics and logs, you’ll be able to diagnose issues more quickly, track system performance, and ensure the operator is running reliably in production.
+
+## Metrics
+
+The ngrok Kubernetes Operator exposes a set of Prometheus-compatible metrics that provide insight into the operator’s internal state and the resources it manages. By enabling metrics, you can monitor reconciliation activity, track resource health, and build alerts around key operational behaviors. 
+
+### Supported metrics
+
+This project is built using Kubebuilder; out of the box it exposes the metrics listed [here](https://book.kubebuilder.io/reference/metrics-reference.html?highlight=metrics#default-exported-metrics-references)
+
+### Accessing metrics
+This operator exposes prometheus metrics on the `/metrics` endpoint. The metrics are exposed on the `:8080` port and can be scraped by prometheus or other services using typical means.
+
+## Logging
+
+The ngrok Kubernetes Operator generates structured logs that capture its reconciliation processes, configuration changes, and any errors encountered while managing resources. These logs are a primary source of truth when diagnosing issues or understanding the operator’s actions.
+
+### Configure log levels
+Lorem ipsum
```

Comment:
https://github.com/ngrok/ngrok-operator/blob/main/helm/ngrok-operator/values.yaml#L210-L220

Reply:
https://github.com/ngrok/ngrok-operator/blob/main/helm/ngrok-operator/templates/controller-deployment.yaml#L129-L130

Reply:
https://ngrok-iambmelt-k8s-153.mintlify.app/getting-started/kubernetes/ingress#install-the-operator

`--set log.level=<whatever> // sets whatever log level`